src/equations/polytropic_euler_2d.jl

```diff
@@ -301,6 +301,46 @@ end
     return abs(v1) + c, abs(v2) + c
 end
 
+# Calculate maximum wave speed for local Lax-Friedrichs-type dissipation as the
+# maximum velocity magnitude plus the maximum speed of sound
+@inline function max_abs_speed_naive(u_ll, u_rr, orientation::Integer,
+                                     equations::PolytropicEulerEquations2D)
+    rho_ll, v1_ll, v2_ll = cons2prim(u_ll, equations)
+    rho_rr, v1_rr, v2_rr = cons2prim(u_rr, equations)
+
+    # Get the velocity value in the appropriate direction
+    if orientation == 1
+        v_ll = v1_ll
+        v_rr = v1_rr
+    else # orientation == 2
+        v_ll = v2_ll
+        v_rr = v2_rr
+    end
+    # Calculate sound speeds (we have p = kappa * rho^gamma)
+    c_ll = sqrt(equations.gamma * equations.kappa * rho_ll^(equations.gamma - 1))
+    c_rr = sqrt(equations.gamma * equations.kappa * rho_rr^(equations.gamma - 1))
+
+    λ_max = max(abs(v_ll), abs(v_rr)) + max(c_ll, c_rr)
+end
+
+@inline function max_abs_speed_naive(u_ll, u_rr, normal_direction::AbstractVector,
+                                     equations::PolytropicEulerEquations2D)
+    rho_ll, v1_ll, v2_ll = cons2prim(u_ll, equations)
+    rho_rr, v1_rr, v2_rr = cons2prim(u_rr, equations)
+
+    # Calculate normal velocities and sound speed (we have p = kappa * rho^gamma)
+    # left
+    v_ll = v1_ll * normal_direction[1] +
+           v2_ll * normal_direction[2]
+    c_ll = sqrt(equations.gamma * equations.kappa * rho_ll^(equations.gamma - 1))
+    # right
+    v_rr = v1_rr * normal_direction[1] +
+           v2_rr * normal_direction[2]
```

Comment:
```suggestion
    v_ll = (v1_ll * normal_direction[1] +
            v2_ll * normal_direction[2])
    c_ll = sqrt(equations.gamma * equations.kappa * rho_ll^(equations.gamma - 1))
    # right
    v_rr = (v1_rr * normal_direction[1] +
            v2_rr * normal_direction[2])
```
I believe for safety we need to wrap the line break with parentheses.

Reply:
Good catch. I remember you learned this the hard way with a MHD specific extension of the AnalysisCallback

Reply:
I think this is not necessary as the `+` operator requires an `lvalue`, correct? I think I even suggested the simplification we now revert :D

parantheses are actually required for 

```julia
c = a 
   + b
```

while 
```julia
c = a +
    b
```

Should give the desired behaviour.

Funny (or sad) enough, I think nearly everyone I talked to who is Julia runs into this issue because the first version I posted does not throw a warning or anything.

Sadly, the main developers seem not to care too much: https://github.com/JuliaLang/julia/issues/33859

Reply:
Yes, that's why I'd prefer to add parens such that if another user modifies these lines in the future, they cannot accidentally produce an incorrect result.

Reply:
Hmm, that is somewhat annoying. I still vote to put in parentheses for the sake of explicitness and/or consistency.

Reply:
> I still vote to put in parentheses for the sake of explicitness and/or consistency.

👍

Reply:
I agree on the parentheses. Will add them in future PRs. For me it should be ready to merge.